NGCHM/WebContent/javascript/Linkout.js

```diff
@@ -914,8 +914,61 @@ NgChm.createNS('NgChm.LNK');
 		const { plugin, params } = pluginData[nonce];
 		pluginData[nonce].params = options;
 		loc.paneTitle.innerText = plugin.name + '. ' + options.plotTitle;
-		NgChm.LNK.initializePanePlugin (nonce, options);
-	};
+		if (plugin.config.axes[0].hasOwnProperty('extra')) {

```

Comment:
What should be the key name used for indicating there are additional covariates needed for the plugin?